heatmap/src/heatmap.rs

```diff
@@ -42,27 +33,22 @@ where
     /// be slightly longer than the requested span. Smaller durations for the
     /// resolution cause more memory to be used, but a smaller batches of
     /// samples to age out at each time step.
-    pub fn new(
-        max: Value,
-        precision: u8,
-        span: Duration<Nanoseconds<u64>>,
-        resolution: Duration<Nanoseconds<u64>>,
-    ) -> Self {
+    pub fn new(m: u32, r: u32, n: u32, span: Duration, resolution: Duration) -> Self {
```

Comment:
Could you add some more detailed explanation as to what these parameters mean to the doc comment for this method? Something like
```
- `m` - does this
- `r` - does that
- etc...
```
as is, I have no idea what `m`, `r`, or `n` mean.

Reply:
Good reminder that this needs a builder and a copy paste of the doc from ccommon.

Reply:
Fixed